### PR TITLE
Added npm packaging for geometry hpp

### PR DESCRIPTION
**Pull request by flippmoke**
As per conversion with @springmeyer adding npm packaging for geometry hpp for use in node cpp applications.

### CODE REVIEW

**Comment by jfirebaugh** [comment]
@flippmoke @springmeyer Can you elaborate on the rationale here? Why not use mason?

**Comment by springmeyer** [comment]
@jfirebaugh thanks for asking. The usecase here is for writing node c++ addons and easily integrating c++ header-only libs without adding extra complexity that is usual to node.js devs or extra dependencies (like mason). By duel publishing in npm it makes it possible to include something like `geometry.hpp` in the `dependencies` section of the `package.json` and the `include_dirs` section of `binding.gyp` in the same, consistent way that https://github.com/nodejs/nan does it. More details at https://github.com/mapbox/cpp/issues/20

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+General guidelines for contributing to mapnik-vector-tile
```
Comment (mourner) [inline CONTRIBUTING.md]: Not `mapnik-vector-tile`.

Comment (mourner) [inline CONTRIBUTING.md]: not mapnik-vector-tile

**Comment by springmeyer** [comment]
@flippmoke let's abandon adding a package.json here per https://github.com/mapbox/cpp/issues/20#issuecomment-272044708 (However a contributing is still well worth it).

**Pull request closed without merging**